code/data_preprocessing/genotype/PCA.ipynb

```diff
@@ -865,7 +865,8 @@
     "    if(length(unique(pheno$ID))!=length(pheno$ID)) stop(\"There are duplicated names in IID column of phenoFile\")\n",
     "\n",
     "    pca <-merge(pheno, pca,by =\"ID\", all=FALSE)\n",
-    "    dat$pc_scores = bind_rows(dat$pc_scores, pca)\n",
+    "    int = intersect(dat$pc_scores%>%colnames,pca%>%colnames )\n",
```

Comment:
This also does not look correct -- the original PCA results and the projected PCs should have the same dimension. Doesnt the loading matrix from the `flashpca` analysis have `K` columns?